iosApp/iosApp/TestbedViewController.swift

```diff
@@ -269,7 +310,48 @@ class TestbedViewController: UIViewController {
         }
         return (UserCommand(rawValue: command!), input)
     }
-
+    
+    private func buildOktaAuthorizeUrl() -> String {
+        guard let plistPath = Bundle.main.path(forResource: "Okta", ofType: "plist"),
+                let plistData = FileManager.default.contents(atPath: plistPath),
+                let plistDictionary = try? PropertyListSerialization.propertyList(from: plistData, options: [], format: nil) as? [String: Any],
+                let oktaDomain = plistDictionary["oktaDomain"] as? String,
+                let clientId = plistDictionary["clientId"] as? String,
+                let signInRedirectURI = plistDictionary["signInRedirectURI"] as? String,
+                let scope = plistDictionary["scopes"] as? String,
+                let oktaState = plistDictionary["oktaState"] as? String,
+                let codeChallengeMethod = plistDictionary["codeChallengeMethod"] as? String,
+                let codeChallenge = plistDictionary["codeChallenge"] as? String
+        else {
+            fatalError("Unable to read Okta.plist or missing required key")
```

Comment:
I see `fatalError` is being used here and below. `fatalError` will stop execution of the app too. Is that the intention, or is there a way we can report the error and not have to halt the app.

Reply:
You are right. I will print this error to the screen instead.